src/main/java/seedu/module/model/task/Module.java

```diff
@@ -60,6 +61,27 @@ public static boolean isSupportedModuleCode(String test) {
         return existingModules.contains(test);
     }
 
+    public int getWorkloadCount() {
+        return workloadCount;
+    }
```

Comment:
hmm correct me if im wrong, but youll be adding the raw workload of each task to the module? Maybe, as a better indicator for modules, we can calculate the proportion / fraction of the total workload for each module? I think that wll be a more useful number for the user? Maybe the rest can pitch in here too

Reply:
I've changed it to show how many tasks there are across all 3 workload ratings for a single module.

Reply:
cool! Wld be good too if u can include a total workload count for each module? Thought that'd be a nice thing to have.

Reply:
Just to clarify, it's just re-add the original workload count implementation? 

So we have:
-One variable counting how many low workload tasks there are
-One for medium workload tasks
-One for high workload tasks
-One that sums up the workload rating translated to int